contrib/kube-local/README.md

```diff
@@ -0,0 +1,393 @@
+# Welcome to kube-local documentation page
+
+- [Welcome to kube-local documentation page](#welcome-to-kube-local-documentation-page)
+  * [About](#about)
+  * [Demos](#demos)
+  * [Supported distros](#supported-distros)
+  * [Supported runtime](#supported-runtime)
+  * [Supported container orchestration](#supported-container-orchestration)
+  * [How it works?](#how-it-works-)
+  * [Examples](#Examples)
+  * [Running](#running)
+    + [Manual run](#manual-run)
+    + [Automatic run](#automatic-run)
+  * [About kube-local.profile](#about-kube-localprofile)
+    + [General options](#general-options)
+    + [Autoinstall](#autoinstall)
+    + [Logging](#logging)
+    + [Operational System](#operational-system)
+    + [Golang](#golang)
+    + [Runtime](#runtime)
+    + [Runtime Client](#runtime-client)
+    + [Runtime Monitor](#runtime-monitor)
+    + [Container Orchestration](#container-orchestration)
+    + [Container Network](#container-network)
+    + [Additional info for the cluster](#additional-info-for-the-cluster)
+
+## About
+kube-local is a build tool helper based on [local-up-cluster.sh](https://github.com/kubernetes/kubernetes/blob/master/hack/local-up-cluster.sh). It helps developers set their local Kubernetes environment as traditional installer or automated.
+
+**Use cases**:
+   - Setup development environment
+   - Matrix testing
+   - Continuous Integration (CI)
+   - Compare outputs from different components
+   - *[Add your use case here....]*
+
+## Demos
+
+### Demo 1 - Auto-install in quiet mode and execute kubectl commands
+Auto-installing kubernetes and it's requirements with CRI-O as runtime, conmon as runtime monitor and container network with network plugins.
+[![asciicast](https://asciinema.org/a/307812.png)](https://asciinema.org/a/307812)
```

Comment:
At some point in time, it might be good to save and run these ascinema's from https://cri-o.io/ as we do the Podman ascinema's in [podman.io](https://github.com/containers/podman.io/tree/master/asciinema).

Reply:
Thanks Tom.  Let's hear from the others members. I don't have perms for the website.